Client/Frontend/Browser/Browser.swift

```diff
@@ -243,7 +243,7 @@ class Browser: NSObject {
         helperManager!.addHelper(helper, name: name)
     }
 
-    func getHelper(#name: String) -> BrowserHelper? {
+    func getHelper(name name: String) -> BrowserHelper? {
```

Comment:
Nit: Can we rename this method to helperWithName and remove the explicit named parameter to make it more swifty?